app/src/main/java/edu/uc/reedws/musiclink/SearchActivity.kt

```diff
@@ -3,7 +3,7 @@ package edu.uc.reedws.musiclink
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import kotlinx.android.synthetic.main.main_activity.*
+import kotlinx.android.synthetic.main.main_activity.libraryButton
```

Comment:
I removed the ' * ' from the imports because it is not a best practice to import all the imports under the library even if you don't need it. Let's say there is a java.awt.Event import which you didn't need but you still imported like this: java.awt.*. Now if you have your own Event import: reedws.edu.Event that you want to import, you won't be able to call Event object without writing the whole import location, which will lead to clutter in your codebase.

You can also refer this community answer to understand more about import '*' issue here: https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad